app/src/androidTest/java/org/mozilla/focus/activity/helpers/EspressoHelper.java

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.activity.helpers;
+
+import android.support.test.espresso.InjectEventSecurityException;
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.view.View;
+
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+
+public class EspressoHelper {
```

Comment:
\[PMD\] INFO: The class 'EspressoHelper' has a Cyclomatic Complexity of 8 \(Highest = 7\)\.